packages/utils/src/toKeySet.test.ts

```diff
@@ -0,0 +1,99 @@
+import { KeyChanges } from "./partitionByKey"
+import { TestScheduler } from "rxjs/testing"
+import { toKeySet } from "./toKeySet"
+import { asapScheduler, map, observeOn, of, Subject } from "rxjs"
+
+const scheduler = () =>
+  new TestScheduler((actual, expected) => {
+    expect(actual).toEqual(expected)
+  })
+
+describe("toKeySet", () => {
+  it("transforms key changes to a Set", () => {
+    scheduler().run(({ expectObservable, cold }) => {
+      const expectedStr = "                     xe--f-g--h"
+      const source$ = cold<KeyChanges<string>>("-a--b-c--d", {
```

Comment:
Silly, but let's make sure that errors get propagated (you will see why in the next comment :see_no_evil:)
```suggestion
      const expectedStr = "                     xe--f-g--h#"
      const source$ = cold<KeyChanges<string>>("-a--b-c--d#", {
```